articles/tables.md

```diff
@@ -246,8 +239,8 @@ In case of a conflict, your update may be rejected. You then have to do another
 # Change the album value 'Vol. 2' to 'Volume 2' 
 results = syn.tableQuery("select * from %s where album='Vol. 2'" %table.schema.id)
 df = results.asDataFrame()
-df['album'] = ['Volume 2']
-syn.store(Table(schema, df, etag=results.etag))
+df['album'] = 'Volume 2'
+syn.store(Table(results, df))
```

Comment:
Confusing naming of `results`. Also, why do you need to create a new table to update?